src/test/java/co/rsk/federate/signing/hsm/client/HSMResponseHandlerV1Test.java

```diff
@@ -36,7 +36,7 @@ public void createResponseHandler() {
     @Test(expected = HSMAuthException.class)
     public void validateResponseRejectedKeyError() throws HSMClientException {
         ObjectNode response = new ObjectMapper().createObjectNode();
-        response.put("errorcode", -1);
+        response.put("errorcode", -103);
```

Comment:
Did the response codes change for HSM v1?

Reply:
Yes, -666 Is No Longer The Code For That Particular Error, It Wasn't Updated All This While

Reply:
Resolved